training/COMPARISON.md

```diff
@@ -0,0 +1,462 @@
+# Arkouda vs NumPy/Pandas
+This document compares and contrasts Arkouda to the similar well known libraries, `NumPy` and `Pandas`. As you go through this document, we encourage you to try the code for yourself. Especially, in the final section.
+
+<a id="toc"></a>
+## Table of Contents
+1. [Background Information](#bg)
+2. [Importing Arkouda](#import)
+3. [Disconnecting From Arkouda Server](#discon)
+4. [Arrays](#arrays)
+   1. [Creating Arrays in NumPy](#nparray_create)
+   2. [Creating Arrays in Arkouda](#akarray_create)
+   3. [Indexing](#ind)
+   4. [Set Operations](#setops)
+   5. [GroupBy](#groupby)
+   6. [Not Supported](#no_support)
+5. [Dataframes](#df)
+   1. [Creating Pandas DataFrame](#df_pd_create)
+   2. [Creating Arkouda DataFrame](df_ak_create)
+   3. [GroupBy on DataFrame](#df_groupby)
+6. [Example Application](#ex)
+
+<a id="bg"></a>
+## Background Information
+
+Arkouda began with the intention of allowing `NumPy` like functionality in an HPC setting. Overtime, it has evolved to support `Pandas` functionality and continues to expand. In order to make Arkouda approachable, the API is designed in way that mimics `NumPy` and `Pandas`. This allows for an easy transition for users that are already familiar with the Python packages, but want to run against larger datasets.
+
+So, what makes Arkouda special? The answer is simple, scalability. Where other packages run against compiled C/C++ code, Arkouda leverages Chapel. This allows Arkouda to run locally, but also allows it to gain significant performance benefits in an HPC setting. 
```

Comment:
the second is not an independent clause because it's missing the subject (there's an implied this). I agree with your rephrasing! I think that sounds good

Reply:
Updated to the rephrasing.